### PR TITLE
Update guide.md

### PR DESCRIPTION
**Pull request by k0pernicus**
Add a comment to know how to pass arguments via Cargo

### CODE REVIEW

**Comment by alexcrichton** [comment]
Thanks for the PR! In this case the `--` isn't actually needed, that's generally only necessary if more flags are being passed to the command being invoked.

**Comment by k0pernicus** [comment]
Hey! Thank you for the review? I will modify the change, and make a new PR ;-)

File: src/doc/guide.md [hunk 1]
```diff
@@ -92,6 +92,8 @@ class="s1">     Fresh</span> hello_world v0.1.0 (file:///path/to/project/hello_w
 class="s1">   Running</span> `target/debug/hello_world`
 Hello, world!</code></pre>
 
+To pass some arguments to your program, you can use `cargo run first_argument second_argument`. If flags are being passed to the command being invoked, please to use `cargo run -- --a_flag -b an_argument`.
```
Comment (alexcrichton) [inline src/doc/guide.md]: The second sentence here may want to be worded along the lines of:

> If flags are being passed to the executable being run a `--` separator can be used to tell Cargo which flags go where, for example: `cargo run -- --foo -b bar`.

Also, can you be sure to wrap this line to 80 chars?

Comment (k0pernicus) [inline src/doc/guide.md]: Sure, I'll try ;-)

Thanks a lot!

**Pull request merged**